docs/introduction.md

```diff
@@ -1,42 +1,43 @@
 # Introduction
 
-deck.gl was created to make "reactive" WebGL-based visualizations of
-"big data" simple, and to enable new users to quickly get impressive
-results, while gradually enabling more flexibility and power as
-users climb the learning curve.
+Deck.gl is a WebGL-powered framework that features an instancing and layering
```

Comment:
As a first sentence, this is a little more correct but also a little more complicated...